### DISCUSSION TITLE
OKX,CoinExに入金を送れない

### DISCUSSION THREAD

**Discussion by Taruto1959** [Chia-Network/chia-blockchain#17230] [Support]
OKI,CoinExのアドレス
送るを押します 。保留中のままで遅れません。
量 0.5 XCH
バージョン 2.1.3
プラットフォーム　　ウィンドウズ11
UIモード　　　GUI (日本語)
解決方法を教えてください。

**Comment by BrandtH22** [comment]
@Taruto1959  , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

The chain has seen high utilization leading to fee pressure so any transactions submitted without fees are taking lower priority than those which have fees. Many users have been able to resolve their pending fees by cancelling the previous transaction onchain and resubmitting it with a higher fee. 

We have created this guide of standard transaction types and their minimum effective fees (minimum meaning that time sensitive transactions may require more in fees to be processed accordingly):
https://docs.chia.net/coin-set-costs/#estimated-transaction-costs

Let us know if you have any questions or run into issues when resubmitting the transaction :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
